docs/release-notes/release-notes-0.15.0.md

```diff
@@ -67,6 +67,11 @@
 * Improved instructions on [how to build lnd for mobile](https://github.com/lightningnetwork/lnd/pull/6085).
 * [Log force-close related messages on "info" level](https://github.com/lightningnetwork/lnd/pull/6124).
 
+## Monitoring
+
+A new [flag (`--prometheus.perfhistogram`) has been added to enable export of
```

Comment:
nit: flag is actually plural. And not sure if parenthesis within a link work or need to be escaped as `\(`?

Reply:
Fixed, and looks to work in the final rendered output.